lib/internal/console/constructor.js

```diff
@@ -509,12 +509,12 @@ const consoleMethods = {
   },
 };
 
-// Returns true if label was not found
+// Returns true if label was found
 function timeLogImpl(self, name, label, data) {
   const time = self._times.get(label);
-  if (!time) {
+  if (time === undefined) {
```

Comment:
You can use typeof here.

Reply:
Yea. I don't think there is any benefit to using it here though.